release_notes/Service_Fabric_ReleaseNotes_81.md

```diff
@@ -0,0 +1,111 @@
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | **Preview** Add support for .NET 6.0 Service Fabric applications | **Brief desc**: Service Fabric version 8.1 introduces programming model support for .NET 6.0 Service Fabric applications.<br> |
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | Add API support application description update | **Brief desc**: Service Fabric version 8.1 introduces REST API support for updating Service Fabric applications. Customer can specify application name, which is a required parameter, flags, which denote which of the parameters will be updated, maximum nodes, minimum nodes, application metrics and whether application capacities should be removed.<br>Corresponding PowerShell API is documented here: https://docs.microsoft.com/en-us/powershell/module/servicefabric/update-servicefabricapplication?view=azureservicefabricps.<br>**Usage**: Added UpdateApplication REST API<br>**Impact**: Customer is able to update application with REST API.<br> |
```

Comment:
Customers can specify an application name and denote application parameters to be updated. These parameters include the maximum nodes, minimum nodes, application metrics and whether application capacities should be removed.

Reply:
done